.devcontainer/devcontainer.json

```diff
@@ -30,6 +30,10 @@
 	"remoteEnv": {
 		"GOFLAGS": "-buildvcs=false"
 	},
+	// When files change, get the latest tools and re-run envtest setup to get the latest
+	// Kubernetes version. The version is extracted from the "ENVTEST_K8S_VERSION = x.y.z"
+	// line in the Makefile.
+	"updateContentCommand": "make kustomize controller-gen envtest && bin/setup-envtest use $(grep -E '^ENVTEST_K8S_VERSION[[:space:]]*=[[:space:]]*[[:digit:][:punct:]]*' Makefile | cut -d'=' -f2 | tr -d '[:space:]') --bin-dir /workspaces/kumquat/bin",
```

Comment:
Just as question why do we need this?

Reply:
Good question! The silly answer is that these were listed as dependencies of the `test` target in the `Makefile`, so I copied them here 😉 

The `envtest` target is needed because it downloads the `bin/setup-envtest` file.

The call to `bin/setup-envtest` is what downloads the control plane binaries and puts them in `bin/k8s/…`. Those are needed for the integration tests to run.

I’m not 100% certain that `kustomize` and `controller-gen` are needed. `controller-gen` does make some source files that start with `zz_`, but we already have them in the source tree, and should always keep them up to date if we do things like change the definition of our `Template`. I can try removing them and re-test in a clean dev container, what do you think @amirbavand ?

Reply:
@amirbavand I checked and neither the `kustomize` nor `controller-gen` targets were necessary to run the integration tests, so I’ve removed them. Thanks!